src/Database/PostgreSQL/Consumers/Config.hs

```diff
@@ -21,12 +22,16 @@ data Action
   | RerunAfter Interval
   | RerunAt UTCTime
   | Remove
-    deriving (Eq, Ord, Show)
+  deriving (Eq, Ord, Show)
 
 -- | Result of processing a job.
 data Result = Ok Action | Failed Action
   deriving (Eq, Ord, Show)
 
+-- | The mode the consumer will run in.
+data Mode = Standard | Duplicating (RawSQL ())
```

Comment:
There's zero documentation on behavior of `Duplicating` mode, which at this moment, is vastly different from `Standard` and pretty much ignores the `run_at` field.

What should happen when you have duplicate jobs and one is scheduled for immediate execution while few more are scheduled in the future? What exactly should the de-duplicating logic do in different cases?